client/src/app/core/core-services/member.service.ts

```diff
@@ -74,4 +111,41 @@ export class MemberService {
             follow: [{ idField: 'committee_ids' }, { idField: 'is_present_in_meeting_ids' }]
         };
     }
+
+    private async doDeleteByPresenter(users: ViewUser[]): Promise<void> {
+        const data = await this.presenter.call<GetUserRelatedModelsPresenterResult>(Presenter.GET_USER_RELATED_MODELS, {
+            user_ids: users.map(user => user.id)
+        });
+
+        for (const user of users) {
+            data[user.id].name = user.getFullName();
+        }
+
+        const promptDialog = this.dialog.open(MemberDeleteDialogComponent, { ...mediumDialogSettings, data });
+        if (await promptDialog.afterClosed().toPromise()) {
+            await this.doDelete(users);
+        } else {
+            throw new NoActionError();
```

Comment:
"No Action" Should not be an error, but is perfectly normal behavior. Don't just throw an error here.